sway-core/src/build_config.rs

```diff
@@ -314,12 +311,12 @@ impl BuildConfig {
         }
     }
 
-    pub fn with_experimental(self, experimental: ExperimentalFlags) -> Self {
-        Self {
-            experimental,
-            ..self
-        }
-    }
+    // pub fn with_experimental(self, experimental: ExperimentalFeatures) -> Self {
```

Comment:
Same question going through my head :smile: I am curious why not a part of `BuildConfig`. But if it isn't lets delete the commented code.

Reply:
Done.